DESCRIPTION.md

```diff
@@ -7,6 +7,8 @@ https://docs.snowflake.com/
 Source code is also available at: https://github.com/snowflakedb/snowflake-connector-python
 
 # Release Notes
+- v3.13.0(TBD)
+  - Improved `write_pandas` and `BindUploadAgent.upload` to create scoped temporary object instead of temporary object.
```

Comment:
why do we need this change in the public connector?
I thought "scoped object" only applied to the concept of stored proc. is this just for aligning the change with the sproc connector?

Reply:
here is some context: https://docs.google.com/document/d/1qfJevIMEZL7O12H5pQK5WZG-a-uwA2K2cmFDusHjdXM/edit?usp=sharing.
I think mostly they want temp object being created as job-scoped object, and they also want to align with sp connector

Reply:
Is my understanding correct?: for regular / non-SP session,  "job" is equivalent to the session itself. So job-scoped objects are de facto session-scoped ones. Therefore this change does not affect behavior of regular sessions.